label_sync/labels.yaml

```diff
@@ -0,0 +1,355 @@
+#    name: area/node-lifecycle
+#  - color: 0052cc
+#    name: area/nodecontroller
+#  - color: d4c5f9
+#    name: area/os/coreos
+#  - color: d4c5f9
+#    name: area/os/fedora
+#  - color: d4c5f9
+#    name: area/os/gci
+#  - color: d4c5f9
+#    name: area/os/ubuntu
+#  - color: d4c5f9
+#    name: area/platform/aws
+#  - color: d4c5f9
+#    name: area/platform/azure
+#  - color: d4c5f9
+#    name: area/platform/gce
+#  - color: d4c5f9
+#    name: area/platform/gke
+#  - color: d4c5f9
+#    name: area/platform/mesos
+#  - color: d4c5f9
+#    name: area/platform/vagrant
+#  - color: d4c5f9
+#    name: area/platform/vsphere
+#  - color: 0052cc
+#    name: area/release-infra
+#  - color: 0052cc
+#    name: area/reliability
+#  - color: 0052cc
+#    name: area/rkt
+#  - color: 0052cc
+#    name: area/secret-api
+#  - color: d93f0b
+#    name: area/security
+#  - color: 0052cc
+#    name: area/stateful-apps
+#  - color: 0052cc
+#    name: area/swagger
+#  - color: 0052cc
+#    name: area/system-requirement
+#  - color: 0052cc
+#    name: area/teardown
+#  - color: 0052cc
+#    name: area/test
+#  - color: 0052cc
+#    name: area/test-infra
+#  - color: 0052cc
+#    name: area/third-party-resource
+#  - color: 0052cc
+#    name: area/ui
+#  - color: 0052cc
+#    name: area/upgrade
+#  - color: 0052cc
+#    name: area/usability
+#  - color: 0052cc
+#    name: area/workload-api/cronjob
+#  - color: 0052cc
+#    name: area/workload-api/daemonset
+#  - color: 0052cc
+#    name: area/workload-api/deployment
+#  - color: 0052cc
+#    name: area/workload-api/job
+#  - color: 0052cc
+#    name: area/workload-api/replicaset
+  - color: d93f0b
+    name: beta-blocker
+  - color: fef2c0
+    name: cherrypick-approved
+  - color: fef2c0
+    name: cherrypick-candidate
+  - color: bfe5bf
+    name: 'cla: human-approved'
+  - color: e11d21
+    name: 'cla: no'
+  - color: bfe5bf
+    name: 'cla: yes'
+  - color: e11d21
+    name: 'cncf-cla: no'
+  - color: bfe5bf
+    name: 'cncf-cla: yes'
+  - color: e11d21
+    name: do-not-merge
+  - color: fbca04
+    name: flake-has-meta
+  - color: 006b75
+    name: for-new-contributors
+  - color: 006b75
+    name: help-wanted
+  - color: fbca04
+    name: keep-open
+#  - color: c7def8
+#    name: kind/api-change
+#  - color: e11d21
+#    name: kind/bug
+#  - color: c7def8
+#    name: kind/cleanup
+#  - color: c7def8
+#    name: kind/design
+#  - color: c7def8
+#    name: kind/documentation
+#  - color: c7def8
+#    name: kind/enhancement
+#  - color: c7def8
+#    name: kind/feature
+#  - color: f7c6c7
+#    name: kind/flake
+#  - color: c7def8
+#    name: kind/friction
+#  - color: f7c6c7
+#    name: kind/mesos-flake
+#  - color: c7def8
+#    name: kind/new-api
+#  - color: c7def8
+#    name: kind/old-docs
+#  - color: bfe5bf
+#    name: kind/postmortem
+#  - color: eb6420
+#    name: kind/support
+#  - color: c7def8
+#    name: kind/technical-debt
+#  - color: fbca04
+#    name: kind/upgrade-test-failure
```

Comment:
OK, bikeshed time!  The full list of `kind/*` labels is gratuitous, but it would be nice if the `/kind` command was useful across all kubernetes repos. 

I propose that the following subset get uncommented:
- `kind/bug`
- `kind/documentation`
- `kind/feature`